lib/identity_cache/parent_model_expiration.rb

```diff
@@ -28,6 +29,22 @@ def install_pending_parent_expiry_hooks(model)
         end
       end
 
+      def parent_expiration_entries
+        ParentModelExpiration.install_pending_parent_expiry_hooks(cached_model)
+        _parent_expiration_entries
+      end
+
+      def check_for_unsupported_parent_expiration_entries
+        return unless parent_expiration_entries.any?
+        msg = "Unsupported manual expiration of record embedded in parent associations:\n"
```

Comment:
@Hectorhammett will this be needed for your use case?

Reply:
This should not cause any problems with our use.